packages/web-components/fast-foundation/src/checkbox/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from "./checkbox.template";
-export * from "./checkbox";
+export * from "./checkbox.template.js";
```

Comment:
Is it possible to have TypeScript files instead of JavaScript?

Reply:
TS doesn't convert the .ts extensions to .js when it does the build, so you have to use the .js extension, but it's actually a TS file.